### ISSUE TITLE
Provide OSGi Meta-information

### ISSUE THREAD

**Issue by laeubi** [danekja/jdk-serializable-functional#7]
To make it easier to use this lib in OSGi-Env it would be cool to add OSGi-Metainfos to the Manifest. This can be done manually or automatic e.g. with http://njbartlett.name/2015/03/27/announcing-bnd-maven-plugin.html

Since this meta-infos are only added to the manifest and no other dependecies are introduced this can be done without influence other usages.

This relates to https://issues.apache.org/jira/browse/WICKET-6402 / https://ops4j1.jira.com/browse/PAXWICKET-486

**Comment by danekja** [comment]
Hello, that seems like a very good idea. I'm going to transform the library into OSGi bundle in some foreseeable future

**Comment by laeubi** [comment]
Sounds great :+1: 
Let me know if you need some help.

**Comment by danekja** [comment]
So, finally got around other to stuff to do this. 

@laeubi could you please check the output of current master build whether it contains valid manifest with all the required information? I will deploy new version to central afterwards.

**Comment by danekja** [comment]
Actually, here is the manifest for review :)

```
Manifest-Version: 1.0
Bnd-LastModified: 1508858849483
Bundle-Contributors: Emond Papegaaij,Jezza
Bundle-Description: A library containing Serializable versions of functi
 onal interfaces from java.util.function package.
Bundle-Developers: Jakub Danek <danek.ja@gmail.com>
Bundle-License: MIT
Bundle-ManifestVersion: 2
Bundle-Name: serializable-function
Bundle-SymbolicName: serializable-function
Bundle-Version: 1.8.4.201710241527
Created-By: 1.8.0_151 (Oracle Corporation)
Export-Package: org.danekja.java.misc.serializable;version="1.8.4";uses:
 ="org.danekja.java.util.function.serializable",org.danekja.java.util.fu
 nction.serializable;version="1.8.4"
Originally-Created-By: 1.8.0_151 (Oracle Corporation)
Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
Tool: Bnd-2.4.1.201501161923

```

**Comment by laeubi** [comment]
Looks good to me.

**Comment by danekja** [comment]
Done and released as 1.8.4.
